README.md

```diff
@@ -114,6 +114,27 @@ a working system.
 
 **Note:** If every role does not go green in `pod-status --watch` refer to [Troubleshooting](#troubleshooting)
 
+3. Pulling updates
+
+    When you want to pull the latest changes from the upstream you should:
+
+    ```
+    # Pull the changes (or checkout the commit you want):
+    git pull
+
+    # Update all submodules to match the checked out commit
+    git submodules update --recursive
```

Comment:
Sometimes `git submodule sync --recursive` is necessary as well. (And sometimes we need to alternate the two commands until they unblock each other).